minutes/20200213.md

```diff
@@ -0,0 +1,78 @@
+|Corda Network Foundation|[Document history]({{ site.github.repository_url }}/blame/master/{{page.path}})|
+
+Minutes of the Board Meeting of 6 December 2019
```

Comment:
Something is not right, the file is `20200213`, but the meeting in the description is on _6 December 2019_.